rolling-shutter/cmd/chain/init.go

```diff
@@ -73,6 +83,8 @@ func initCmd() *cobra.Command {
 	cmd.PersistentFlags().String("listen-address", "tcp://127.0.0.1:26657", "tendermint RPC listen address")
 	cmd.PersistentFlags().String("role", "validator", "tendermint node role (validator, isolated-validator, sentry, seed)")
 	cmd.PersistentFlags().Uint64("initial-eon", 0, "initial eon")
+	cmd.PersistentFlags().Int64("forks.check-in-update.height", 0, "block height at which to activate the check-in update fork")
+	cmd.PersistentFlags().Bool("forks.check-in-update.disabled", false, "whether the check-in update fork is disabled")
```

Comment:
Double negatives can be confusing. Maybe `enabled` defaulting to `true` would be clearer.